managers/views.py

```diff
@@ -186,8 +186,8 @@ def get_context_data(self, **kwargs: Any) -> dict:
         context["task_activities"] = TaskActivityType.objects.filter(is_active=True)
         return context
 
-    def form_valid(self, form: TaskActivityForm) -> HttpRequest:
-        form.save()
+    def form_valid(self, form: TaskActivityForm) -> HttpResponse:
+        TaskActivityType(name = self.request.POST["name"])
```

Comment:
I think that you can omit this line.  `form_valid()` function will be helpful in validation input and resolving issue with creating the same task activities that I mentioned.

Reply:
changed